packages/millicast-sdk/src/config.js

```diff
@@ -1,5 +1,5 @@
 const config = {
-  MILLICAST_DIRECTOR_ENDPOINT: 'https://director.millicast.com',
+  MILLICAST_DIRECTOR_ENDPOINT: 'https://director-dev.millicast.com',
```

Comment:
This change is causing the E2E tests to fail, as the credentials used are for prod environment. I suggest to undo this, as it is important to test that environment with the SDK.